README.md

```diff
@@ -1,4 +1,4 @@
-StatsdBundle [![Build Status](https://travis-ci.org/M6Web/StatsdBundle.png?branch=master)](https://travis-ci.org/M6Web/StatsdBundle)
+StatsdBundle [![Build Status](https://github.com/BedrockStreaming/StatsdBundle/actions/workflows/tests.yml/badge.svg)](https://github.com/BedrockStreaming/StatsdBundle/actions/workflows/tests.yml)
```

Comment:
the image URL does not look like the one that was used [here](https://github.com/BedrockStreaming/Tornado/pull/50/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R4) 

but I don't know if it's a big deal 🤷

Reply:
It seems to be the Github official way to add a badge now.